webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -46,16 +52,27 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
         this.setState({playing: false});
     };
 
+    toggleShortsExpanded = () => {
+        this.setState({shortsExpanded: !this.state.shortsExpanded});
+    };
+
     render() {
         const {metadata, link} = this.props;
 
         const videoId = getVideoId(link);
+        const isShorts = getIsShortsVideoLink(link);
+        const videoType = isShorts ? messages.shorts : messages.youtube;
         const videoTitle = metadata?.title || 'unknown';
         const time = handleYoutubeTime(link);
 
         const header = (
             <h4>
-                <span className='video-type'>{'YouTube - '}</span>
+                <span className='video-type'>
+                    <FormattedMessage
+                        {...videoType}
+                        values={{type: videoType}}
+                    />
+                </span>
```

Comment:
The way you have this implemented right now, you wouldn't need a value here.

My idea regarding the translation was also to add the url in the translation, so the translators have more context and, if necessary, they could change the order or add other limiters. So... having the message as...

`<type>Youtube - </type>{title}`

And here do something like:
```
<FormattedMessage
    {...videoType}
    values={{
        title: (
            <span className='video-title>
                ...
            </span>
         ),
         type: (v) => (<span className='video-type>{v}</span>),
     }}
/>
```